src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,45 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    // Test Case 구현
+    @Test
+    @DisplayName("Set의 크기 확인 테스트")
+    void setSizeTest() {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @ParameterizedTest(name = "contains 메소드 테스트 - {0} 포함 테스트")
+    @ValueSource(ints = {1, 2, 3})
+    void setContainsTest(int number) {
+        assertThat(numbers.contains(number)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(value);
```
참고차ㅎㅎ 위와 같은 방법도 가능해요!

Reply:
이번 테스트 연습 과제에 있던 내용인데 깜빡하게 된것같습니다... 감사합니다!!